jwst/datamodels/steppars.py

```diff
@@ -0,0 +1,34 @@
+"""Step parameters model"""
+from copy import copy
+
+from .model_base import DataModel
+
+__all__ = ['StepParsModel']
+
+DEFAULT_META = {
+    'date': 'SPECIFY DATE',
+    'origin': 'STScI',
+    'telescope': 'JWST',
+    'reftype': 'pars-step',
+    'pedigree': 'SPECIFY PEDIGREE',
+    'description': 'Parameters for calibration step SPECIFY',
+    'author': 'SPECIFY AUTHOR',
+    'useafter': 'SPECIFY',
+    'instrument': {
+        'name': 'SPECIFY',
+    },
+}
+
+
+class StepParsModel(DataModel):
+    """
+    A data model for `Step` parameters.
+    """
+    schema_url = "steppars.schema"
+    supported_formats = ['yaml', 'json', 'asdf']
```

Comment:
What does `supported_formats` do?  And does the datamodel actually have a serializer that can write these out as JSON or YAML?

Reply:
Absolutely nothing apparently. Will remove.